### PR TITLE
Add Events Helpers to PDI Connector

### PR DESCRIPTION
**Pull request by IanRFerguson**
Adding helpers written by @cmdelrio to the `Events()` object. These functions are inherited by the `PDI()` connector and will be available post-authentication

### CODE REVIEW

**Review by cmdelrio — APPROVED** [review]
lgtm!

**Comment by IanRFerguson** [comment]
@shaunagm I've tested this locally and it's returning data in the anticipated format, let me know if you have any questions

**Comment by IanRFerguson** [comment]
Marking this as "medium priority" as it's blocking a sync-in from Mobilize

File: parsons/pdi/events.py [hunk 1]
```diff
@@ -506,3 +506,14 @@ def update_activity_assignment(
         )
 
         return response
+
+    def get_event_activity_assignments(self, start_date, end_date, expand, limit=None):
```
Comment (shaunagm) [inline parsons/pdi/events.py:510]: Limit is not getting used here?

**Review by shaunagm — CHANGES_REQUESTED** [review]
These need docstrings, otherwise they won't appear in the documentation. They should also have automated tests written for them. If it's important enough you can merge as-is and do a follow-up PR with docs and tests but be forewarned I will hound you for it ;)

File: parsons/pdi/events.py [hunk 2]
```diff
@@ -506,3 +506,81 @@ def update_activity_assignment(
...
+                Per the API docs, use "YYYY-MM-DD" format
+
+            end_date: str
+                Latest records to be returned in the API response.
+                Per the API docs, use "YYYY-MM-DD" format
+
+            limit: int
+                Specify limit to return (max=2000)
+
+        `Returns`:
```
Comment (Jason94) [inline parsons/pdi/events.py:560]: Missing `Returns` doc

Comment (IanRFerguson) [reply parsons/pdi/events.py:560]: thanks Jason!

**Review by shaunagm — APPROVED** [review]
Looks great, thanks for adding the docs & tests

**Pull request merged**